packages/webpack/src/utils/style-loader.js

```diff
@@ -12,8 +13,15 @@ export default class StyleLoader {
     this.perfLoader = perfLoader
     this.resolveModule = resolveModule
 
-    if (buildContext.options.build.postcss) {
-      this.postcssConfig = new PostcssConfig(buildContext)
+    const { postcss: postcssOptions } = buildContext.options.build
+    if (postcssOptions) {
+      const postcss = require(resolveModule('postcss'))
```

Comment:
Is this OK to use `require(resolveModule(...))` ?

Reply:
Seems good 👍